app/code/community/Drip/Connect/Model/Observer/Wishlist/RemoveProductsWithoutQuantity.php

```diff
@@ -21,22 +21,19 @@ protected function executeWhenEnabled($observer)
             return;
         }
 
-        //loop through each product and check quantity
-        if (filter_input_array(INPUT_POST) &&
-            (null !== filter_input(INPUT_POST, 'description')) &&
-            is_array(filter_input(INPUT_POST, 'description'))) {
-            foreach (filter_input(INPUT_POST, 'description') as $itemId => $description) {
+        $post_description = filter_input(INPUT_POST, 'description', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY);
```

Comment:
Nit: Magento standard is camel case
```suggestion
        $postDescription = filter_input(INPUT_POST, 'description', FILTER_DEFAULT, FILTER_REQUIRE_ARRAY);
```